dashboard/scripts/import_hint_prompt_threshold.rb

```diff
@@ -30,6 +30,34 @@
 # where quartile = 4
 # group by 1;
 
+# UPDATES for 2020: script names and date ranges updated
```

Comment:
can we delete the query above since the query has been updated? or do we need both? if we need both, i think we should add comments to explain when/why you'd use one vs the other